client/src/components/OnePlace/index.js

```diff
@@ -1,13 +1,29 @@
 import React, { Component } from 'react'
-import ListOfItems from '../ListOfItems'
+import './style.css'
 
 class OnePlace extends Component {
   render() {
+    const { place } = this.props
+    console.log('in one', place)
     //  this.props.match.url.split(":")[1])
-    const x = this.props.listOfItems
+    // const x = this.props.listOfItems
     return (
       <div>
-        <p>one place item</p>
+        <div>
+          <img className="palceImage1Details" src={`${place.image1}`} />
```

Comment:
I don't think you need to add place in the css names here as it's not a reusable component.